csrc/preseg_passes/translate_no_reduction_matmul_to_mul_squeeze.cpp

```diff
@@ -166,32 +167,21 @@ class NoReductionMatmulToMulSqueezeTranslator {
 
       if (matrix_ndims_a == 2 && matrix_ndims_b == 1) {
         // Case 1
-        if (std::any_of(bc_flags_a.begin(), bc_flags_a.end(), [](bool flag) {
```

Comment:
IIUC, when B is 1D, there should be no missing batch dimensions, hence no broadcast required for A. So this can be safely removed.